products/kdeplasma.md

```diff
@@ -75,6 +75,6 @@ every other year. There is a [detailed schedule](https://community.kde.org/Sched
 future bug fix releases and major releases.
 
 ## Release Cadence
-* Plasma 5.27 and 5.24 LTS critical bug fix support will end once the first Plasma 6.0 feature release comes out.
+* Plasma 5.27 and 5.24 critical bug fix support will end once the first Plasma 6.0 feature release comes out.
```

Comment:
According to https://community.kde.org/Schedules/Plasma_5#LTS_releases 5.24 LTS release will receive bugfix updates until we start Plasma 6 development. And 5.27 at least until Plasma 6.0 is released according to https://community.kde.org/Schedules/Plasma_5#Support_status_by_Release_Series.

Reply:
I think it's OK to drop the 'LTS' label. But the issue remain.

You wrote _Plasma 5.27 and 5.24 critical bug fix support will end once the first Plasma 6.0 feature release comes out._, but I think that's wrong for 5.24.

According to https://community.kde.org/Schedules/Plasma_5#LTS_releases 5.24 LTS release will receive bugfix updates only until they start Plasma 6 development. Given the 5.27 will probably be the latest LTS (from what I understood) the 5.24 should be EOL soon (but it's not EOL yet).

Reply:
Okay, I see your point. Do you agree with the following:
```suggestion
* Plasma 5.24 LTS will get critical bug fixes updates until start of Plasma 6 development (timetable unsure). The last Plasma 5 release will be version 5.27 LTS and the end of critical bug fixes is not yet announced.
```

Reply:
Yep, with a few adjustments (will commit them directly).